graphics/QtPBFImagePlugin/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           qmake5 1.0
 
-github.setup        tumic0 QtPBFImagePlugin 1.4
+github.setup        tumic0 QtPBFImagePlugin 2.0
 revision            1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
Fixed.